packages/app/templates/ui-extensions/projects/pos_ui/shopify.ui.extension.toml.liquid

```diff
@@ -1,5 +1,6 @@
 type = "{{ type }}"
 name = "{{ name }}"
+description = "{{ name }}"
```

Comment:
Wondering if this description is a placeholder which is why it is using `name`? I thought it would be something like this:
```
description = "{{ description }}"
```